detection_rules/schemas/__init__.py

```diff
@@ -256,15 +256,15 @@ def load_stack_schema_map() -> dict:
 @cached
 def get_stack_schemas(stack_version: Optional[str] = '0.0.0') -> OrderedDictType[str, dict]:
     """Return all ECS + beats to stack versions for every stack version >= specified stack version and <= package."""
-    stack_version = Version(stack_version or '0.0.0')
-    current_package = Version(load_current_package_version())
-
-    if len(current_package) == 2:
-        current_package = Version(current_package + (0,))
+    if stack_version and stack_version != "0.0.0" and len(stack_version.split(".")) == 2:
```

Comment:
Should we compare using versions here instead of strings?

Reply:
Honestly, we don't need the check at all since we have the `optional_minor_and_patch` flag.